core/chains/evm/logpoller/log_poller.go

```diff
@@ -412,11 +412,11 @@ func (lp *logPoller) backfill(ctx context.Context, start, end int64) error {
 // 1. Find the LCA by following parent hashes.
```

Comment:
I see a few more places in this file, which still depend on HeaderByNumber which should also be changed to HeadByNumber.

Reply:
yeah those ones aren't critical, but I'll adjust them for consistency